src/rdo.rs

```diff
@@ -280,10 +273,13 @@ fn compute_tx_rd_cost(
       }
     }
   }
-  // Compute rate
-  let rate = (bit_cost as f64) / ((1 << OD_BITRES) as f64);
+  distortion
+}
 
-  (distortion as f64) + lambda * rate
+fn compute_rd(fi: &FrameInvariants, rate: u32, distortion: u64) -> f64 {
```

Comment:
Ok, let be brave to ask full names for someone's sakes (including me)
compute_rd --> compute_rd_cost